docs/GettingStartedDocs/install_oe_sdk-Ubuntu_22.04.md

```diff
@@ -0,0 +1,88 @@
+# Install the Open Enclave SDK (Ubuntu 22.04)
+
+## Platform requirements
+
+- Ubuntu 22.04-LTS 64-bit.
+- SGX1-capable system with support for Flexible Launch Control (FLC).
+    - You can acquire a VM with the required features from [Azure Confidential Compute](https://azure.microsoft.com/en-us/solutions/confidential-compute/).
+    - If you are setting up your own device, [check if your existing device supports SGX with FLC](/docs/GettingStartedDocs/Contributors/building_oe_sdk.md#1-determine-the-sgx-support-level-on-your-developmenttarget-system).
+        - If your device only supports SGX without FLC, you will need to [clone and build OE SDK](/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md) for that configuration.
+        - If your device does not support SGX, follow the [instructions for simulation mode](/docs/GettingStartedDocs/install_oe_sdk-Simulation.md). Please see [instructions for determining SGX support](/docs/GettingStartedDocs/SGXSupportLevel.md) on the machine you are using.
+
+### 1. Configure the Intel and Microsoft APT Repositories
+```bash
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+
+echo "deb [arch=amd64] https://packages.microsoft.com/ubuntu/22.04/prod focal main" | sudo tee /etc/apt/sources.list.d/msprod.list
+wget -qO - https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+
+sudo apt update
+```
+
+### 2. Install the Intel SGX DCAP Driver
+Some versions of Ubuntu come with the SGX driver already installed. You can check
+by running with the following:
+
+```
+$ dmesg | grep -i sgx
+[  106.775199] sgx: intel_sgx: Intel SGX DCAP Driver {version}
+```
+
+If the output of the above is blank, you should proceed with installing the driver:
+
+```bash
+sudo apt update
+sudo apt -y install dkms
+wget https://download.01.org/intel-sgx/sgx-linux/2.25/distro/ubuntu22.04-server/sgx_linux_x64_driver_1.41.bin -O sgx_linux_x64_driver.bin
```

Comment:
Is this the version available inbox in Ubuntu 22.04?

Reply:
Though not explicitly verified, v1.41 is the only version available for Ubuntu 22.04